src/components/SortButton/SortButton.jsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '@mui/material';
+
+const SortButton = ({ sortOrder, onSortChange }) => {
```

Comment:
Don't pass the setter functions to other components. This is giving the `SortButton` component too much control over the state of the parent.

Instead, pass a single `toggleSort` function as a prop that does not take any arguments